test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -456,31 +449,18 @@ def setUpClass(cls):
         cls.init_param()
         cls.check_users_exist()
         cls.init_servers()
-        cls.init_schedulers()
```

Comment:
I'm not sure I understand why you made changes to this file, can you please clarify?

Reply:
it was a merge issue. I restored the file to what is in master because no changes are necessary.